src/main/java/com/researchspace/webapp/controller/repositories/DigitalCommonsDataUIConnectionConfig.java

```diff
@@ -0,0 +1,70 @@
+package com.researchspace.webapp.controller.repositories;
+
+import static com.researchspace.service.IntegrationsHandler.DIGITAL_COMMONS_DATA_APP_NAME;
+
+import com.researchspace.model.User;
+import com.researchspace.model.oauth.UserConnection;
+import com.researchspace.properties.IPropertyHolder;
+import com.researchspace.service.UserConnectionManager;
+import com.researchspace.webapp.integrations.digitalcommonsdata.DigitalCommonsDataController;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.security.Principal;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.cxf.common.security.SimplePrincipal;
```

Comment:
We're not importing org.apache.cxf.* anyhwere in rspace-web codebase, so would be good if we can avoid it here.

Reply:
removed